utils/config/config.go

```diff
@@ -572,6 +573,7 @@ type ServerDetails struct {
 	ArtifactoryUrl                  string `json:"artifactoryUrl,omitempty"`
 	DistributionUrl                 string `json:"distributionUrl,omitempty"`
 	XrayUrl                         string `json:"xrayUrl,omitempty"`
+	XscUrl                          string `json:"xscUrl,omitempty"`
```

Comment:
Why did you add that if it is never used?
at client-go you are using GetUrl() and you are setings using the value at `XrayUrl`

Reply:
we use this field for logic in security-cli